README.rst

```diff
@@ -96,6 +96,30 @@ To disable an existing MFE, set its corresponding configuration setting to "null
     tutor config save --set MFE_GRADEBOOK_MFE_APP=null
     tutor config save --set MFE_PROFILE_MFE_APP=null
 
+Adding custom translations to your MFEs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This plugin makes it possible to customise existing translations, and to add custom translation to MFEs. Here is how to do it:
+
+1. Identify the ID of the string you would like to translate. For instance, the ID of the "Account Information" string in the account MFE is "account.settings.section.account.information" (see `source <https://github.com/edx/frontend-app-account/blob/1444831833cad4746b9ed14618a499b425ccc907/src/account-settings/AccountSettingsPage.messages.jsx#L34>`__).
+2. Create a folder and i18n file corresponding to your MFE app and language in the Tutor root. This location of this file should be ``/path/to/tutor/env/plugins/mfe/build/mfe/i18n/<app name>/<language code>.json``. For instance, to add French ("fr") translation strings to the account MFE, run:
+
+    cd "$(tutor config printroot)/env/plugins/mfe/build/mfe/i18n/"
+    mkdir account
+    touch account/fr.json
+
+3. Add your entries to this file in JSON format, where the key is the string ID and the value is the actual string. For instance:
+
+    {
+      "account.settings.section.account.information": "Information du compte POUAC"
+    }
+
+4. Rebuild the MFE image and restart the MFE with:
+
+    tutor images build mfe
+    tutor local start -d
+
+Your custom translation strings should now appear in your app.
```

Comment:
> For how long? Do you need to redo this customisation operation every time you update the underlying code?

Once your translation strings are stored in this folder, they will be baked in the Docker container every time it is built (and thus every time it is started, because the MFE container is built before starting). I guess I should add this precision to the readme.

> How does this interact with translations from Transifex, which are pulled down automatically/regularly and checked into master in the MFE? Is this an overlay?

Yes, exactly. The i18n-merge.js script basically overrides the strings from the src/messages folder.

Reply:
> Thinking about how this fits into our customization story. Is this intended to be _the_ method by which operators can change strings if they need to adjust them?

Yes, when they deploy their MFE with this plugin. The only alternative is to fork the upstream repo, which is really not desirable.

> What's the use case for adding new strings?

Sometimes, strings are missing from the messages files. This might be the case if the release branch was created before the up-to-date strings are pulled from Transifex.

Reply:
> I guess I should add this precision to the readme.

Yes, I think that would be good.

@davidjoy a major use case for this which I've heard is locale-specific translations.

Reply:
@sarina Naive question, perhaps, but how does that differ from the translations done in Transifex?  Aren't they "locale-specific"?

Reply:
Well think about British English - "torch" instead of "flashlight", "organisation" vs "organization", etc. People want a site that actually looks like their locale. Spanish is very different in Spain vs Lat Am, and even _within_ LatAm, there are different colloquialisms and idioms in each country.

Reply:
And, we don't really have much `en_uk` or `es_AR` translations. It doesn't make sense, effort-wise, to retranslate when 90+% of the translations will remain the same. There's no "copy all the translations from one language into a new one" functionality in Transifex.

Reply:
Ah, okay, different "extended language subtags", which I had to look up just now to remind myself of the naming, but which is what the second part is. :D  Yeah... our system isn't smart enough to first fall back to the more general primary language subtags before falling back to the default language.  If it were, then this would have less utility.  But I getcha. 👍🏻